ddtrace/internal/flare/_subscribers.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime
+import os
+from typing import Callable  # noqa:F401
+from typing import Optional  # noqa:F401
+
+from ddtrace.internal.flare.flare import Flare
+from ddtrace.internal.logger import get_logger
+from ddtrace.internal.remoteconfig._connectors import PublisherSubscriberConnector  # noqa:F401
+from ddtrace.internal.remoteconfig._subscribers import RemoteConfigSubscriber
+
+
+log = get_logger(__name__)
+
+DEFAULT_STALE_FLARE_DURATION_MINS = 20
+
+
+class TracerFlareSubscriber(RemoteConfigSubscriber):
+    def __init__(
+        self,
+        data_connector: PublisherSubscriberConnector,
+        callback: Callable,
+        flare: Flare,
+        stale_flare_age: int = DEFAULT_STALE_FLARE_DURATION_MINS,
+    ):
+        super().__init__(data_connector, callback, "TracerFlareConfig")
+        self.current_request_start: Optional[datetime] = None
+        self.stale_tracer_flare_num_mins = stale_flare_age
+        self.flare = flare
+
+    def has_stale_flare(self) -> bool:
+        if self.current_request_start:
+            curr = datetime.now()
+            flare_age = (curr - self.current_request_start).total_seconds()
+            stale_age = self.stale_tracer_flare_num_mins * 60
+            return flare_age >= stale_age
+        return False
+
+    def _get_data_from_connector_and_exec(self):
+        if self.current_request_start is not None and self.has_stale_flare():
```

Comment:
```suggestion
        if self.has_stale_flare():
```

I think this is the same thing, since you check `if self.current_request_start:` and return False if it is